packages/react/src/components/TimePicker/TimePicker.types.ts

```diff
@@ -60,32 +64,43 @@ export interface ITimePickerProps
   allowFreeform?: boolean;
 
   /**
-   * Custom time range to for time options
+   * Custom time range to for time options.
    */
   timeRange?: ITimeRange;
 
   /**
-   * Localized strings to use in the TimePicker
+   * Localized strings to use in the TimePicker.
    */
   strings?: ITimePickerStrings;
 
   /**
-   * Default value of the TimePicker, if any
+   * The uncontrolled default selected time.
    */
   defaultValue?: Date;
 
   /**
-   * Callback issued when the time is changed
+   * A Date representing the selected time. If you provide this, you must maintain selection
+   * state by observing onChange events and passing a new value in when changed.
```

Comment:
```suggestion
   * state by observing onChange events and passing a new value in when changed.
   * mutually exclusive with `defaultValue`.
```

Reply:
Resolved by updating comment to suggested.